surveys/static/js/charts.js

```diff
@@ -517,3 +534,15 @@ function binValue(value, bins) {
   }
   return binnedValue;
 }
+
+/**
+ * Shuffles array in place. ES6 version
+ * @param {Array} a items An array containing the items.
+ */
+function shuffle(a) {
+    for (let i = a.length - 1; i > 0; i--) {
+        const j = Math.floor(Math.random() * (i + 1));
+        [a[i], a[j]] = [a[j], a[i]];
+    }
+    return a;
+}
```

Comment:
At this point `let` and `const` are pretty darn close to full browser support with [~94% of the market](https://caniuse.com/#search=let), but they're still not quite universally supported. Since we don't have a transpiler set up for this app, let's go with the safer `var`.